src/main/resources/schema/schema_v9.sql

```diff
@@ -0,0 +1,9 @@
+ALTER TABLE system_table
+  ADD COLUMN langtags TEXT ARRAY;
+
+CREATE TABLE system_settings (
+  key   VARCHAR(255) PRIMARY KEY,
+  value TEXT
+);
+
+INSERT INTO system_settings (key, value) VALUES ('langtags', $$["de-DE", "en-GB"]$$);
```

Comment:
Better idea? Defaults are defaults and I would use `de-DE` and `en-GB` as defaults.